rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date, :book, :person
+  attr_reader :rentals
+
+  def initialize(book, person, date)
+    @book = book
+    @person = person
+    @date = date
+    book.rentals = self
+    person.rentals = self
```

Comment:
- [ Required ] The `book.rentals` defined in class `Book` and `person.rentals` defined in class `Person` are arrays and so the association won't work correctly if you set them to `self`. I would like to suggest that you push rental object `self` into the `book.rentals` and `person.rentals` array. Please see the example below and rectify this.

```ruby
  def initialize(book, person, date)
    @book = book
    @person = person
    @date = date
    book.rentals << self
    person.rentals << self
  end
```